src/main/scala/edu/berkeley/ce/sparkrocks/RockSlicer.scala

```diff
@@ -68,7 +68,8 @@ object RockSlicer {
       broadcastJoints.value.zipWithIndex.foldLeft(Seq(seedBlock)) { case (currentBlocks, (joint, idx)) =>
         if (idx % REDUNDANT_ELIM_PERIOD == 0) {
           // When idx is a multiple of REDUNDANT_ELIM_PERIOD, check for geometrically redundant faces
-          currentBlocks.flatMap(_.cut(joint, generation=idx)).map { case block @ Block(center, _, generation) =>
+          currentBlocks.flatMap(_.cut(joint, arguments.minRadius, arguments.maxAspectRatio,
+            generation=idx)).map { case block @ Block(center, _, generation) =>
```

Comment:
You mean just in terms of formatting or am I missing something? I've changed it based on what I think you mean, but let me know if I misinterpreted!